Modules/ITS/src/ITSClusterTask.cxx

```diff
@@ -97,7 +97,18 @@ void ITSClusterTask::initialize(o2::framework::InitContext& /*ctx*/)
 
   getJsonParameters();
 
-  o2::base::GeometryManager::loadGeometry(mGeomPath.c_str());
+  if (mLocalGeometryFile == 1) {
+    ILOG(Info, Support) << "Getting geometry from local file" << ENDM;
+    o2::base::GeometryManager::loadGeometry(mGeomPath.c_str());
+  } else {
+    ILOG(Info, Support) << "Getting geometry from ccdb - timestamp: " << mGeoTimestamp << ENDM;
+    auto& mgr = o2::ccdb::BasicCCDBManager::instance();
+    mgr.setTimestamp(mGeoTimestamp);
+    mgr.get<TGeoManager>("GLO/Config/GeometryAligned");
+    if (!o2::base::GeometryManager::isGeometryLoaded()) {
+      ILOG(Error, Support) << "Can't retrive geometry from ccdb: " << mgr.getURL() << " timestamp: " << mGeoTimestamp << ENDM;
```

Comment:
can it survive such a condition? what will happen later ? should it be a fatal and it exits ?